.github/workflows/ci_bionic.yml

```diff
@@ -0,0 +1,48 @@
+name: CI - Ubuntu Bionic
+
+on:
+  # direct pushes to protected branches are not supported
+  pull_request:
+  # run every day, at 6am UTC
+  schedule:
+    - cron: '0 6 * * *'
+  # allow manually starting this workflow
+  workflow_dispatch:
+
+jobs:
+  industrial_ci:
+    name: ROS Melodic (${{ matrix.ros_repo }})
+    runs-on: ubuntu-20.04
+
+    strategy:
+      fail-fast: false
+      matrix:
+        ros_distro: [ melodic ]
+        ros_repo: [ main, testing ]
+
+    env:
+      CCACHE_DIR: "${{ github.workspace }}/.ccache"
+      CATKIN_LINT: "true"
```

Comment:
This shouldn't need quotes, but it's not important.

Reply:
it was copied from `fanuc`

Reply:
I know. It needs to be removed there as well.